provider/src/test/kotlin/org/gradle/kotlin/dsl/integration/KotlinBuildScriptModelIntegrationTest.kt

```diff
@@ -186,7 +186,7 @@ class KotlinBuildScriptModelIntegrationTest : AbstractIntegrationTest() {
             rootProjectScript = "",
             subProjectScript ="""
                 buildscript {
-                    dependencies { classpath(kotlin("gradle-plugin")) }
+                    dependencies { classpath(kotlin("gradle-plugin", "$embeddedKotlinVersion")) }
```

Comment:
:x: We should introduce a `DependencyHandler.embeddedKotlin(module: String)` dependency builder to replace the old usages.

Reply:
Done